docs/stratis.txt

```diff
@@ -66,6 +68,15 @@ filesystem rename <pool_name> <fs_name> <new_name>::
 blockdev list [pool_name]::
 	 List all blockdevs that make up the specified pool, or all pools, if
 	 no pool name is given.
+key set <(--keyfile-path | --capture-key)> <key_desc>::
```

Comment:
Wherever there's the flag ```--keyfile-path``` it should also include the ```<keyfile_path>``` argument that it must take.

Reply:
Oops! You're right.